### ISSUE TITLE
Expand documentation on authentication with cookies

### ISSUE THREAD

**Issue by kevinmamaqi** [directus/docs#260]
I am developing an app with Directus as a Headless CMS and ReactJS as the front-end. I am having difficulties to understand how to store a Cookie so authenticated users can access to the dashboard and their information. I use AXIOS to authenticate and it works well so far:
```
axios
    .post("http://localhost:9932/_/auth/authenticate", {
        email: email,
        password: password
        // mode: "cookie"
    })
    .then(response => {
        if (response.status === 200) {
            console.log("ha funcionado");
            console.log(response);
        }
    })
    .catch(error => {
        console.log("axios error", error);
    });
```
The problem is that I receive the same data using or not the `mode: cookie` and that the documentation doesn't expand much on this topic. I found this https://github.com/directus/api/wiki/Session-Auth-Flow - which seems that the cookie authentication is a work in progress and that in order to be able to use authentication with directus I will need the JS-SDK to store the JWT token. Is that correct?

Moreover, is there an endpoint where I can check if the user is logged in with sending the credentials (Cookie)?.

**Comment by rijkvanzanten** [comment]
@kevinmamaqi The authenticate request with mode: cookie will set a httpOnly cookie in the browser on authentication. This is invisible from JavaScript for security reasons. 

> Moreover, is there an endpoint where I can check if the user is logged in with sending the credentials (Cookie)?.

The API will return `public: true` in the JSON if the request wasn't authenticated. I normally fetch the `/` endpoint to check for that public flag. Alternatively, you can fetch `/users/me`. If that returns a 404, you're not logged in.

**Comment by kevinmamaqi** [comment]
Hi @rijkvanzanten 

How do I set up that cookie with an axios request for example? I cannot see the Cookie when inspecting with Google Chrome on the Application TAB and the request returns the TOKEN.

**Comment by rijkvanzanten** [comment]
Use the `mode` attribute when using the `/auth/authenticate` endpoint:

https://docs.directus.io/api/authentication.html#mode-optional

With axios directly:

```js
const client = axios.create({
  withCredentials: true,
  baseURL: 'https://example.com/project'
});

client.post('/auth/authenticate', {
  email: 'admin@example.com',
  password: 'password',
  mode: 'cookie'
});
```


```

**Comment by kevinmamaqi** [comment]
Mode cookie definitely doesn't work on my end. I made a new installation cloning from git. Tried to use axios (as I was doing) - It returns the JWT. Then installed sdk-js and used:

client.login({
  url: "https://demo.directus.io",
  email: "admin@example.com",
  password: "password",
  mode: "cookie"
});

It also returns the JWT and other data. But when I try to use 

client.getActivity({
    sort: "action_on"
});

Or collections, etc... I recieve unauthorized access. It does not set up a cookie on my end.
